pyte/escape.py

```diff
@@ -152,3 +152,9 @@
 
 #: *Horizontal position adjust*: Same as :data:`CHA`.
 HPA = "'"
+
+#: *Scroll up*
+SU = "S"
+
+#: *Scroll down*
+SD = "T"
```

Comment:
While the xterm documentation states that 'T' is the correct code for Scroll Down (SD), the xterm code is implemented to handle the incorrect code '^' and treats 'T' as either mouse tracking or SD (depending on the arguments).
I'm not sure how many (or if any) applications still use the incorrect code.